### PR TITLE
Tal r isi sen bug fix master2

### PR DESCRIPTION
**Pull request by talregev**
* Fix app crash on startup. 
* Show the project list menu properly. 
Still not behave properly.
Have crashes and projects not display properly when added. Disappear after you switch a screen.

### CODE REVIEW

File: android/BOINC/app/proguard-rules.txt [hunk 1]
```diff
@@ -21,5 +21,7 @@
 #-renamesourcefileattribute SourceFile
```
Comment (AenBleidd) [inline android/BOINC/app/proguard-rules.txt:21]: Please move these rules files to a separate PR

File: android/BOINC/app/proguard-rules-debug.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Add project specific ProGuard rules here.
```
Comment (AenBleidd) [inline android/BOINC/app/proguard-rules-debug.txt:1]: Please move these rules files to a separate PR

File: android/BOINC/app/build.gradle [hunk 3]
```diff
@@ -70,7 +70,7 @@ android {
             minifyEnabled true
             debuggable true
             testCoverageEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules-debug.txt'
```
Comment (AenBleidd) [inline android/BOINC/app/build.gradle:73]: Please remove this line and make a corresponding change in a separate PR

**Comment by AenBleidd** [comment]
Please change PR title to:
[Component] Description of changes

**Comment by talregev** [comment]
Continue there
https://github.com/BOINC/boinc/pull/3710
